docs/namespaces-per-cluster.md

```diff
@@ -0,0 +1,101 @@
+# Namespaces per cluster Workload
+
+The Namespaces per cluster workload playbook is `workloads/cluster-limits-namespaces-per-cluster.yml` and will run the namespaces per cluster workload on your cluster.
+
+Namespaces per cluster test purpose is a cluster limits focused test which creates maximum possible namespaces across the cluster.
+
+Running from CLI:
+
+```sh
+$ cp workloads/inventory.example inventory
+$ # Add orchestration host to inventory
+$ # Edit vars in workloads/vars/namespaces-per-cluster.yml or define Environment vars (See below)
+$ time ansible-playbook -vv -i inventory workloads/cluster-limits-namespaces-per-cluster.yml
+```
+
+## Environment variables
+
+### PUBLIC_KEY
+Default: `~/.ssh/id_rsa.pub`  
+Public ssh key file for Ansible.
+
+### PRIVATE_KEY
+Default: `~/.ssh/id_rsa`  
+Private ssh key file for Ansible.
+
+### ORCHESTRATION_USER
+Default: `root`  
+User for Ansible to log in as. Must authenticate with PUBLIC_KEY/PRIVATE_KEY.
+
+### WORKLOAD_IMAGE
+Default: `quay.io/openshift-scale/scale-ci-workload`  
+Container image that runs the workload script.
+
+### WORKLOAD_JOB_NODE_SELECTOR
+Default: `true`  
+Enables/disables the node selector that places the workload job on the `workload` node.
+
+### WORKLOAD_JOB_TAINT
+Default: `true`  
+Enables/disables the toleration on the workload job to permit the `workload` taint.
+
+### WORKLOAD_JOB_PRIVILEGED
+Default: `true`  
+Enables/disables running the workload pod as privileged.
+
+### KUBECONFIG_FILE
+Default: `~/.kube/config`  
+Location of kubeconfig on orchestration host.
+
+### PBENCH_SSH_PRIVATE_KEY_FILE
```

Comment:
The vars are a bit mixed up here.  The order for pbench vars should be:
PBENCH_INSTRUMENTATION
ENABLE_PBENCH_AGENTS
ENABLE_PBENCH_COPY
PBENCH_SSH_PRIVATE_KEY_FILE
PBENCH_SSH_PUBLIC_KEY_FILE
PBENCH_SERVER